addNameList/index.js

```diff
@@ -0,0 +1,119 @@
+const name = document.getElementById("name");
+const age = document.getElementById("age");
+const grade = document.getElementById("grade");
+const addDetails = document.getElementById("user-button");
+const display = document.getElementById("display");
+const userCountNode = document.getElementById("total-count");
+
+const errorNametNode = document.getElementById("error-name");
+const errorAgeNode = document.getElementById("error-age");
+const errorGradeNode = document.getElementById("error-grade");
+
+let userCount = 0;
+
+const isValid = (user) => {
+  let error = {};
+  resetErrorField();
+  let flag = 0;
+  if (user.name=="") {
+    error.name = "*Required Field."
+    errorNametNode.innerHTML = error.name;
+    flag = 1;
+  }
+  else if (user.name.length < 3) {
+    error.name = "*Length should be greater then 3."
+    errorNametNode.innerHTML = error.name;
+    flag = 1;
+  }
+  else if (user.name.length > 18) {
+    error.name = "*Length should be less then 18."
+    errorNametNode.innerHTML = error.name;
+    flag = 1;
+  }
+ 
+  if (typeof user.age == "string") {
+    error.age = "kindly put valid digit"
+    errorAgeNode.innerHTML = error.age;
+    flag = 1;
+  }
+  else if (user.age < 0) {
+    error.age = "*Kindly put valid Age."
+    errorAgeNode.innerHTML = error.age;
+    flag = 1;
+  }
+  if(user.grade==""){
+    error.grade = "*Required Field."
+    errorGradeNode.innerHTML = error.grade;
+    flag = 1;
+  }
+  else if (user.grade != "A" && user.grade != "B" && user.grade != "C" && user.grade != "D" && user.grade != "E") {
```

Comment:
this can simplified as
` 
 else if(!["A", "B", "C", "D", "E"].includes(user.grade) ){
 `